### PR TITLE
1921 app base

### PR DESCRIPTION
**Pull request by emitche**
Using the ionic base, create starting base for app. Includes home and login.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Sometimes needed:
+`npm rebuild`
+
+## Run
+`ionic serve`
+
+## Live Reload
+Currently Sass changes are not loading properly in liveReload. You may need to run this command to see your changes:
+
+`gulp sass`
```
Comment (smoyte) [inline README.md]: Nice job on adding a Readme!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# rCredits App
+
+## Environment Setup
+
+### Download the Repo
+
+### Install Packages
+`cd [project]`
+`npm install`
```
Comment (smoyte) [inline README.md]: For blocks like this it's better to use the three backticks. https://help.github.com/articles/markdown-basics/#multiple-lines

File: scss/ionic.app.scss [hunk 3]
```diff
@@ -21,3 +21,6 @@ $ionicons-font-path: "../lib/ionic/fonts" !default;
 // Include all of Ionic
 @import "www/lib/ionic/scss/ionic";
 
+// Application Styles
+@import "www/scss/rcredits";
```
Comment (smoyte) [inline scss/ionic.app.scss]: I think the idea is that you put all of your scss files right in the scss dir, not under www. That may be why live update isn't working. Please see the mealyzer-mobile app as an example. https://github.com/hooverlunch/mealyzer-mobile

File: www/index.html [hunk 4]
```diff
@@ -5,12 +5,13 @@
...
     <title></title>
 
+    <!--
     <link href="lib/ionic/css/ionic.css" rel="stylesheet">
     <link href="css/style.css" rel="stylesheet">
+    -->
 
-    <!-- IF using Sass (run gulp sass first), then uncomment below and remove the CSS includes above
+    <!-- IF using Sass (run gulp sass first), then uncomment below and remove the CSS includes above -->
     <link href="css/ionic.app.css" rel="stylesheet">
```
Comment (smoyte) [inline www/index.html:9]: You can nix the above comments.

File: www/js/app.js [hunk 5]
```diff
@@ -67,7 +67,32 @@ angular.module('starter', ['ionic', 'starter.controllers'])
...
   // if none of the above states are matched, use this as the fallback
-  $urlRouterProvider.otherwise('/app/playlists');
+  $urlRouterProvider.otherwise('/app/rlogin');
+});
+
+app.directive("rcLogo", function(){
+  return {
+    require: "E",
+    templateUrl: "templates/logo.html"
+  }
```
Comment (smoyte) [inline www/js/app.js]: The routes and directives will end up in their own files later but fine for now.

File: www/templates/logo.html [hunk 6]
```diff
@@ -0,0 +1 @@
+<img src="img/rlogo.png" alt="rCredits logo" />
```
Comment (smoyte) [inline www/templates/logo.html:1]: Nice job setting this up as a directive! There will no doubt be tons of these in the app.

For something as simple as a single image, though, a directive might be overkill. I suggest changing this to use ng-include (closer to Rails partials). This is a helpful post: http://stackoverflow.com/a/24172025/2066866.

**Comment by smoyte** [comment]
I tried it out on my machine. Looks good. Just a few comments above and then let's switch off this issue for now. We will need to come back to it later to style the general template a bit more.

File: scss/ionic.app.scss (hunk 3 above)
Comment (emitche) [inline scss/ionic.app.scss]: Moved, but still not fixing live update issue. Mealyzer mobile is using css, not scss it looks like for custom styles.

**Comment by emitche** [comment]
This is updated with requested changes.

**Pull request merged**